tests/foreman/cli/test_host.py

```diff
@@ -66,6 +67,11 @@ def module_default_proxy(module_target_sat):
     return module_target_sat.cli.Proxy.list({'search': f'url = {module_target_sat.url}:9090'})[0]
 
 
+@pytest.fixture(scope='module')
+def module_org(module_target_sat):
+    return module_target_sat.api.Organization(simple_content_access=False).create()
```

Comment:
We can't simply add this to the test as a step, since the test uses another fixture, that calls `module_org`.
We probably have no other choice than wait till https://bugzilla.redhat.com/show_bug.cgi?id=2159974 is fixed,
or go with my solution (disable SCA for all tests in the module) as a temporary measure.

Funny fact: I need this patch to exist in the older branches, where SCA is not even a concern, however i probably won't be able to implement it before it lands here, in `master` branch

Reply:
the bugzilla is now fixed and i managed to control SCA for the org on the test level.